MMM-TeslaFi.js

```diff
@@ -322,5 +359,26 @@ Module.register("MMM-TeslaFi", {
     } else {
       return this.numberFormat(valueMiles) + " miles";
     }
+  },
+
+  // Gets Static map as picture
+  getMap: function (lat, lng) {
+    if (this.config.googleMapApiKey !== "") {
+      const options = {
+        center: [lat, lng],
+        zoom: this.config.mapZoom,
+        key: this.config.googleMapApiKey,
+        marker: [lat, lng]
+      };
+      return `https://maps.googleapis.com/maps/api/staticmap?size=${this.config.mapWidth}x${this.config.mapHeight}&center=${options.center}&markers=${options.marker}&key=${options.key}&zoom=${options.zoom}&size=tiny`;
+    }
+  },
+
+  // Checks exclusion list returns bool
+  isExcluded: function (locale) {
+    const excludeLocationsUpper = this.config.excludeLocations.map((location) =>
+      location.toUpperCase()
+    );
```

Comment:
I know it's only a simple operation, but seems a bit inefficient to do this every time we load the DOM. Perhaps this would be better suited to being in `start()`?

Reply:
Good point, will do!

Reply:
Little lost here - I'm to remove 1 line from the function `isExcluded` and need to, instead, declare a variable at the top of the Module class to persist throughout the whole class - then move that same function to `start()` to update the persistent variable?
Then it'll have to be checked again anyhow.

Is it the code we're cleaning up, or the lifting?

Reply:
Yes, the checking will have to be done every time, but my point was that the transformation of all locations to upper case could be done within `start()`.

Not a huge deal, so I'm happy to leave this as-is if you want?